Library/Formula/sift.rb

```diff
@@ -0,0 +1,38 @@
+require "language/go"
+
+class Sift < Formula
+  desc "Fast and powerful open source alternative to grep"
+  homepage "https://sift-tool.org"
+  url "https://github.com/svent/sift/archive/v0.3.3.tar.gz"
+  sha256 "69b166befdd9a19fd433096fbd9d9ef64b190e7ca6c37af4f6ee058d81ddfd63"
+
+  depends_on "git" => :build
+  depends_on "go" => :build
+
+  go_resource "github.com/svent/go-flags" do
+    url "https://github.com/svent/go-flags.git", :revision => "37c8226983775d404b6edfebd44be1078bd0fe95"
+  end
+
+  go_resource "github.com/svent/go-nbreader" do
+    url "https://github.com/svent/go-nbreader.git", :revision => "7cef48da76dca6a496faa7fe63e39ed665cbd219"
+  end
+
+  bottle do
```

Comment:
Oh, ok. I spent a long time trying to figure that out. I could tell that upgrades were definitely handled automatically, but I didn't see how it would happen initially.